vllm/model_executor/layers/vocab_parallel_embedding.py

```diff
@@ -307,7 +307,7 @@ def forward(self, input_):
         else:
             masked_input = input_
             # Get the embeddings.
-        output_parallel = F.embedding(masked_input, self.weight)
+        output_parallel = F.embedding(masked_input.long(), self.weight)
```

Comment:
Is this needed?

Reply:
seems there are some type issue in xpu torch.jit.script,  expected type is long while xpu version return float. I have reported to ipex team.